CONTRIBUTING.md

```diff
@@ -181,38 +181,31 @@ JavaScript and TypeScript files are validated using a combination of [Prettier](
 To release a stable version, go on `master` (`git checkout master`) and use:
 
 ```sh
-npm run release
+npm run release:prepare
 ```
 
-_Make sure to use `npm run` instead of `yarn run` to avoid issues._
+It will create a pull-request for next release. When it's reviewed, approved and merged, then CircleCI will automatically publish it to NPM.
```

Comment:
```suggestion
It will create a pull request for the next release. When it's reviewed, approved and merged, then CircleCI will automatically publish it to npm.
```